src/webServiceClient.spec.ts

```diff
@@ -1,6 +1,10 @@
+import cloneDeep = require('lodash.clonedeep');
 import nock = require('nock');
-import * as fixtures from '../fixtures/score.json';
+import * as factors from '../fixtures/factors.json';
```

Comment:
LGTM overall. My only comment is the `factors` fixture could just contain `subscores` and get merged onto a copy of the `insights` fixture.

Reply:
This is true.  https://github.com/maxmind/minfraud-api-node/pull/3/commits/48a74cc0bfef662e0e08bfa90a1bb6cd79581dda